mappings/net/minecraft/block/PinkPetalsBlock.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_onsqqphm net/minecraft/block/PinkPetalsBlock
+	FIELD f_crwyarkl MAX_PETALS_AMOUNT I
```

Comment:
the grammar here is a little strange, I'd prefer to go with singular petal for `MAX_PETAL_AMOUNT`
same goes for min petals amount

Reply:
It's changed ^^.